### ISSUE TITLE
Missing function ''euclidean'' and solved. Some questions about the details.

### ISSUE THREAD

**Issue by Alan5279** [yangle15/GraphEmbedding_DR_ELM#1]
I checked the code when I read the paper Graph Embedding-Based Dimension Reduction With Extreme Learning Machine. While running the script "DemoForGDRELM" with Matlab 2019b, IDE shows function or variable ''euclidean'' is missing in line 34. 
The error is located in line 33 of script "GraphGenerate", which is used by script "GDR_ELM" in line 63.
I checked the code, it uses ''feval'' function and calls an function named ''euclidean''. Judging by the name, I assume that the function is to calculate the euclidean distance. So I coded one using expression like: 
dist(i,j) = norm(Block(i,:)-Matrix(j,:), 2);
If you have the same problem, this might help. 

In the meantime, I found that the function calculates the distances between the training data and a block of training data.  So the returning value of the function should be a matrix. However, the dimension seems irrelevant. Either you use the following size:
dist=zeros(Nblock,Ndata);
or you use the other way:
dist=zeros(Ndata, Nblock);
(In this way, the distance calculation becomes:  dist(i,j) = norm(Matrix(i,:)-Block(j,:),2); )
They all work, and the performance results show that the second way of calculation has a higher accuracy (around 5%). I wonder what the differences means to the algorithm, because, to me, I think the first way of calculation makes sense. But the result tells me otherwise.
